src/prog_algs/metrics/toe_profile_metrics.py

```diff
@@ -5,7 +5,7 @@
 """
 from ..predictors import ToEPredictionProfile
 
-def alpha_lambda(toe_profile : ToEPredictionProfile, ground_truth : dict, lambda_value : float, alpha : float, beta : float, **kwargs): 
+def alpha_lambda(toe_profile : ToEPredictionProfile, ground_truth : dict, lambda_value : float, alpha : float, beta : float, **kwargs) -> dict: 
```

Comment:
I noticed the doc comment on line 22 is wrong - can you update that as well to show a dictionary return type